_layouts/default.html

```diff
@@ -8,8 +8,7 @@
 <body id="page-top">
   {% include loader.html %}
   {% include nav.html %}
-  <!-- {% include header.html %} -->
```

Comment:
@alexanderhay2020 this was causing the issue.

Commented liquid include should be removed, otherwise they will be rendered in the page.